ToDoList-frontend/src/app/bucket/bucket.component.ts

```diff
@@ -141,12 +178,16 @@ export class BucketComponent implements OnInit {
       bucketTaskPriority: new FormControl('', [
         Validators.required,
       ]),
+      assigneeControl: new FormControl('', [
+        Validators.required,
+      ])
     });
   }
 
   initializeEditBucketTaskForm() {
     const bucketTaskState = this.bucketTaskStates.find(bts => bts.id === this.currentBucketTask.bucketTaskStateId).name;
     const bucketTaskPriority = this.bucketTaskPriorities.find(btps => btps.id === this.currentBucketTask.bucketTaskPriorityId).name;
+    const assigneeName = this.assignees.find(a => a.id == this.currentBucketTask.assigneeId).name;
```

Comment:
Jesli w wielu miejscach szukamy assignee po id, warto rozwazyc stworzenie osobnej metody `findAssigneeById(id: number)`

Reply:
Zaimplementowałem takie metody:
  findAssigneeById(assignees: Assignee[], assigneeIdKey: number): string {
    return assignees.find(a => a.id == assigneeIdKey).name;
  }

  findAssigneeByName(assignees: Assignee[], assigneeNameKey: string): number {
    return assignees.find(a => a.name === assigneeNameKey).id;
  }
  
Jaki byś dał feedback odnośnie nazw parametrów jakie napisałem?

Reply:
Dałbym po prostu id albo name, nazwa funkcji `findAssigneesById` mowi, ze to, co przekazemy, to bedzie id assignee

Reply:
Poprawione